codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -184,7 +195,8 @@ public final T build() {
          */
         public final T build(Http2Connection connection) {
             Http2FrameReader reader = new DefaultHttp2FrameReader(validateHeaders);
-            Http2FrameWriter writer = new DefaultHttp2FrameWriter();
+            Http2FrameWriter writer = new DefaultHttp2FrameWriter(new DefaultHttp2HeadersEncoder(
```

Comment:
How about adding a new constructor that requires the sensitivity detector to `DefaultHttp2FrameWriter` for simplicity?